counsel.el

```diff
@@ -5222,37 +5222,37 @@ subdirectories that builds may be invoked in."
            counsel-compile-build-directories))
 
 (defun counsel--get-build-subdirs (blddir)
-  "Return all subdirs of BLDDIR sorted by access time."
+  "Return all subdirs of BLDDIR sorted by modification time."
   (mapcar #'car (sort (directory-files-and-attributes
                        blddir t directory-files-no-dot-files-regexp t)
                       (lambda (x y)
-                        ;; FIXME: Access time is NOT the 7th file attribute.
                         (time-less-p (nth 6 y) (nth 6 x))))))
```

Comment:
These should be `(nth 5 ...)`.

Reply:
Hmm nth 5 can't be right - it gets the order wrong and the order by the most recently created on my box.

Reply:
Then you're either not after modification time (which is the 6th file attribute), or you've flipped the arguments to `time-less-p`.

Reply:
(or your file system is falling short of your needs)

Reply:
Oops, you're right. It should be `(nth 6 ...)` because `directory-files-and-attributes` adds the file name at the start of every list element, thus shifting the usual accessors by one.